modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -706,8 +707,129 @@ class SampleCodeTest {
         replicator.start()
         // end::getting-started[]
     }
+    
+    func dontTestPredictiveModel() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
 
+        // tag::register-model[]
+        let model = ImageClassifierModel()
+        Database.prediction.registerModel(model, withName: "ImageClassifier")
+        // end::register-model[]
+        
+        // tag::predictive-query-value-index[]
+        let input = Expression.dictionary(["photo": Expression.property("photo")])
+        let prediction = Function.prediction(model: "ImageClassifier", input: input)
+        
+        let index = IndexBuilder.valueIndex(items: ValueIndexItem.expression(prediction.property("label")))
+        try database.createIndex(index, withName: "value-index-image-classifier")
+        // end::predictive-query-value-index[]
+        
+        // tag::unregister-model[]
+        Database.prediction.unregisterModel(withName: "ImageClassifier")
+        // end::unregister-model[]
+    }
+    
+    func dontTestPredictiveIndex() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
+
+        // tag::predictive-query-predictive-index[]
+        let input = Expression.dictionary(["photo": Expression.property("photo")])
+        let prediction = Function.prediction(model: "ImageClassifier", input: input)
```

Comment:
this seems to be unused?

Reply:
Good catch. Removed in https://github.com/couchbase/docs-couchbase-lite/commit/e894d6dcb59f811992a2ead8db9bd058e664f47a